woocommerce/grow
# Tweak - Upgrade actions/core to v1.10.0

### Changes proposed in this Pull Request:

[Set Output command is being deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) soon and is producing a warning in our JS based actions using SetOutput

For solving it, this PR updates actions/core to v1.10.0 as suggested in the GitHub blog.

Besides that, updating the actions/core package produces an error with rollup build file that is solved by adding `exportConditions: [ 'node' ]` inside the `nodeResolve` plugin to correctly use the node version.


### Detailed test instructions:


1. Confirm that get-plugin-releases action works fine
2. Confirm that get-release-notes action works fine
3.  Confirm that update-version-tags action works fine
4. Confirm that php-diff action works fine
5. Confirm no warning is displayed anymore  (in  update-version-tags  and php-diff) this only happens when errors happen. Since core.setFailed  is called here only)

ℹ️  The release process was tested here: https://github.com/puntope/grow/actions/runs/5055625218/jobs/9072072131
ℹ️  See also https://github.com/puntope/grow/pull/1

### Additional details:


### Changelog entry

> Tweak - Upgrade actions/core to v1.10.0
